services/frontend-service/src/ui/components/ServiceLane/ServiceLane.tsx

```diff
@@ -29,12 +30,12 @@ export const ServiceLane: React.FC<{ application: string }> = (props) => {
                     <Button
                         className="service-action service-action--prepare-undeploy"
                         label={'Prepare to delete'}
-                        icon={'delete'}
+                        icon={<DeleteWhite />}
```

Comment:
just a small hint. it would make this PR much more readable if you split it up.
e.g. do all the images in another PR.
* adding the svgs
* changing the text icons and etc...
* changing the button implementation

Reply:
but for now it's okay

Reply:
Okay, noted!